lib/msal-core/src/utils/RequestUtils.ts

```diff
@@ -75,13 +71,35 @@ export class RequestUtils {
         return validatedRequest;
     }
 
+    /**
+     * 
+     * @param request 
+     * @param clientId 
+     * @param interactionType 
+     */
+    static validateLoginRequest(request: AuthenticationParameters, clientId: string, interactionType: InteractionType): AuthenticationParameters {
```

Comment:
Do we still need a separate validate function for login vs token requests? Since we're always adding `openid` and `profile` to a request is there a difference?

Reply:
Good point, as long as we're using `isLoginCall` to determine the `response_type` for login APIs, there's no reason why the OIDC scopes need to be in the array before `createNavigateURL` is called before the request.